docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,323 @@ task.download_outputs()
 machine_group.terminate()
 ````
 
```

Comment:
Isn't the previous example the same as the foundation? Do the openfoam commands change? If not, then I would remove this example and just talk about the different parameter in the constructor.

Reply:
The commands change. One uses `surfaceFeatures` and the other `surfaceFeatureExtract`

Reply:
Is this the only difference? Then I would refer that, and show which other line changes. Otherwise there is a lot of repetition.